python/cugraph/dask/pagerank/pagerank.py

```diff
@@ -7,13 +7,15 @@
 import os
 from dask.distributed import wait, default_client
 from toolz import first
+import dask.dataframe as dd
 
 
 def to_gpu_array(df):
     """
     Get the gpu_array pointer to the data in columns of the
     input dataframe.
     """
+    df = drop_duplicates(df)
```

Comment:
Can we call `drop_duplicates` in the hibench test instead of here?
I worry that calling it here adds an unnecessary overhead on edge lists that don't have duplicates.

Reply:
edited